sensorprocess/sensorprocess.go

```diff
@@ -20,15 +20,11 @@ type Config struct {
 	CartoFacade            cartofacade.Interface
 	sensorProcessStartTime time.Time
 
-	Lidar                sensors.TimedLidarSensor
-	LidarName            string
-	LidarDataFrequencyHz int
-	currentLidarData     LidarData
+	Lidar            s.TimedLidarSensor
+	currentLidarData LidarData
 
-	IMU                sensors.TimedIMUSensor
-	IMUName            string
-	IMUDataFrequencyHz int
-	currentIMUData     IMUData
+	IMU            s.TimedIMUSensor
+	currentIMUData IMUData
```

Comment:
Refactoring: `Lidar` and `IMU` contain already the lidar/imu name and data frequency information.